pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```
@@ -38,40 +53,33 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
+  ];
 
   strictDeps = true;
 
-  buildPhase = ''
-    runHook preBuild
-
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
+  postPatch = ''
+    lispdir=$out/share/emacs/site-lisp/elpa/cask-${version} \
```

FYI, `finalAttrs` can be used now.

`finalAttrs.version` or `$version` is more correct if `version` is overridden.

```suggestion
    lispdir=$out/share/emacs/site-lisp/elpa/cask-${finalAttrs.version} \
```